data-access-analysis-tools/index-nb-intro.rst

```diff
@@ -76,12 +77,22 @@ All users will find a set of tutorial notebooks provided in the notebooks direct
 Jupyter Notebooks can be identified by their suffix ".ipynb". 
 
 **Using a Terminal:**
-The DP0.1 data sets and the LSST Science Pipelines tools can both be accessed from the command line of a terminal. As described in the default message that appears in all newly-launched terminals, to create a Rubin Observatory environment in a terminal session and set up the full set of packages, users must first execute "source ${LOADSTACK}" and "setup lsst_distrib". For example, to query and retrieve DP0.1 data sets using the Butler (see below), command-line tools are available as documented `here <https://pipelines.lsst.io/v/weekly/modules/lsst.daf.butler/scripts/butler.html>`_. Type "butler --help" in any terminal to see a list of available butler functionality.
+The DP0.1 data sets and the LSST Science Pipelines tools can both be accessed from the command line of a terminal.
+As described in the default message that appears in all newly-launched terminals, to create a Rubin Observatory environment in a terminal session and set up the full set of packages, users must first execute:
+
+.. code-block:: bash
+
+   source ${LOADSTACK}
+   setup lsst_distrib 
+
+For example, to query and retrieve DP0.1 data sets using the Butler (see below), command-line tools are available as documented `here <https://pipelines.lsst.io/v/weekly/modules/lsst.daf.butler/scripts/butler.html>`_.
+Type ``butler --help`` in any terminal to see a list of available butler functionality.
 
 **Safely Log Out of JupyterLab:**
 Use the File item in the top menu bar.
 To safely shut down a Notebook, choose Close and Shutdown Notebook.
```

Comment:
We might want to give users a little advice on whether it's even possible to not save / to revert work that they've done in a notebook session.  I don't fully understand notebook auto-save behavior myself.